src/util/required-opts.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('underscore');
+
+module.exports = function checkAndBuildRequiredOpts (actualOpts, requiredOpts, context) {
+  if (requiredOpts === void 0) {
+    throw new Error('Opts are required');
+  }
+
+  _.each(requiredOpts, function (item) {
+    if (actualOpts[item] === void 0) throw new Error(item + ' is required');
```

Comment:
* I like to add curly braces to every conditional.
* [Same](https://github.com/CartoDB/cartodb.js/pull/1771/files#r141307949)